### ISSUE TITLE
Add binaries to the release

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/OSPSuite-R#1473]
Please add binary packages to the release, so installation is possible without access to GitHub. Also add the required binaries of rSharp, utils, and tlf.

**Comment by Felixmil** [comment]
Binaries were attached to the pre-release: https://github.com/Open-Systems-Pharmacology/OSPSuite-R/releases/tag/v12.1.0

**Comment by Felixmil** [comment]
@PavelBal Can we close this ?

**Comment by PavelBal** [comment]
No ;)

> Also add the required binaries of rSharp, utils, and tlf.

**Comment by Felixmil** [comment]
They are already listed in README: https://github.com/Open-Systems-Pharmacology/OSPSuite-R?tab=readme-ov-file#install-non-cran-dependencies

**Comment by PavelBal** [comment]
Yes, but not the particular versions for this particular release.

**Comment by Felixmil** [comment]
Well this release wasn't tested with the latest **released** versions of these packages but with latest development versions. If we attach the latest dependency binaries and a user install them for using ospsuite, it will not work.
